lms/djangoapps/grades/tests/test_tasks.py

```diff
@@ -159,7 +159,7 @@ def test_query_counts(self, default_store, num_mongo_calls, num_sql_calls, creat
 
     @ddt.data(
         (ModuleStoreEnum.Type.mongo, 1, 24),
-        (ModuleStoreEnum.Type.split, 3, 23),
+        (ModuleStoreEnum.Type.split, 3, 24),
```

Comment:
These counts went up as a result of removing the `pre_publish` signal from `openedx/core/djangoapps/credit/signals.py`. Previously, `is_course_cached` would be cached at pre-publish time, meaning that the same call in `listen_for_grade_calculation` would not touch the database but now does.